src/status_im/contexts/wallet/send/flow_config.cljs

```diff
@@ -18,10 +18,8 @@
     :skip-step? (fn [db] (some? (get-in db [:wallet :ui :send :recipient])))}
    {:screen-id  :screen/wallet.select-asset
     :skip-step? (fn [db] (or (token-selected? db) (collectible-selected? db)))}
-   {:screen-id  :screen/wallet.send-input-amount
-    :skip-step? (fn [db]
-                  (or (not (token-selected? db))
-                      (some? (get-in db [:wallet :ui :send :amount]))))}
+   {:screen-id :screen/wallet.send-input-amount
+    :skip-step? (fn [db] (= (get-in db [:wallet :ui :send :tx-type]) :collectible))}
```

Comment:
Maybe use `(collectible-selected? db)`

Reply:
@briansztamfater I can use it, but why is it's logic that way it is: is there a case where we can have a collectible-stored`:wallet :ui :send :collectible` and the value of `:wallet :ui :send :tx-type` is not `:collectible` ?